test/unit_tests/modules/FlyoutMenu-spec.js

```diff
@@ -28,15 +28,15 @@ describe( 'FlyoutMenu', () => {
   let window;
   let document;
   const HTML_SNIPPET =
-    '<div data-js-hook="behavior_flyout-menu">' +
-      '<button data-js-hook="behavior_flyout-menu_trigger" ' +
-              'aria-pressed="false" ' +
-              'aria-expanded="false"></button>' +
-      '<div data-js-hook="behavior_flyout-menu_content" aria-expanded="false">' +
-        '<button data-js-hook="behavior_flyout-menu_alt-trigger" ' +
-                'aria-expanded="false"></button>' +
-      '</div>' +
-    '</div>';
+    `<div data-js-hook="behavior_flyout-menu">
+      <button data-js-hook="behavior_flyout-menu_trigger"
+              aria-pressed="false"
+              aria-expanded="false"></button>
+      <div data-js-hook="behavior_flyout-menu_content" aria-expanded="false">
+        <button data-js-hook="behavior_flyout-menu_alt-trigger"
+                aria-expanded="false"></button>
+      </div>
+    </div>`;
```

Comment:
This doesn't produce it's own linting errors? We're one version behind on eslint (maybe in the performance sprint—next sprint I'll get to update), and I found that it was complaining about quote styles when template strings were used. If you don't see any new errors this is fine and preferable. If you do it's probably still fine but we'll want to update the eslint rules as soon as possible.

Reply:
Nope, no new errors. @sebworks used this style for the FWB tests and we didn't see any there either.

Reply:
Weird, I saw it somewhere but forget the circumstances. Oh well, all good!